state/in_memory_store.go

```diff
@@ -36,26 +36,21 @@ func NewInMemoryStore(conf *config.Config) *InMemoryStore {
 	}
 }
 
-// GetConfig returns the config
 func (s *InMemoryStore) GetConfig() config.Config {
```

Comment:
relooking at this again, why did we get rid of the comments for these public methods? Shouldn't we always include a comment for a public method? Even if it is part of the interface.

Reply:
Yes, I think you're right. My linter stopped warning me about this for some reason. The comments are a bit redundant but I think it's still the Go standard to have them for public methods. Thank you, adding them back